types/vote_test.go

```diff
@@ -297,22 +309,21 @@ func TestVoteProtobuf(t *testing.T) {
 
 var sink interface{}
 
-var protoVote *tmproto.Vote
-var sampleCommit *Commit
-
-func init() {
-	protoVote = examplePrecommit().ToProto()
+func getSampleCommit(ctx context.Context, t testing.TB) *Commit {
+	t.Helper()
 
 	lastID := makeBlockIDRandom()
-	voteSet, _, vals := randVoteSet(2, 1, tmproto.PrecommitType, 10, 1)
-	commit, err := makeCommit(lastID, 2, 1, voteSet, vals, time.Now())
-	if err != nil {
-		panic(err)
-	}
-	sampleCommit = commit
+	voteSet, _, vals := randVoteSet(ctx, t, 2, 1, tmproto.PrecommitType, 10, 1)
+	commit, err := makeCommit(ctx, lastID, 2, 1, voteSet, vals, time.Now())
+	require.NoError(t, err)
+
+	return commit
 }
 
 func BenchmarkVoteSignBytes(b *testing.B) {
+	protoVote := examplePrecommit(b).ToProto()
+
+	b.ResetTimer()
```

Comment:
Probably strictly speaking want this after the ReportAllocs call (though I think the latter is mostly a flag flip)